tests/jballerina-integration-test/src/test/resources/websocket/wsservices/03_push_and_onText.bal

```diff
@@ -23,7 +23,8 @@ service onTextString on new http:WebSocketListener(9080) {
     resource function onText(http:WebSocketCaller caller, string data, boolean finalFrame) {
         var returnVal = caller->pushText(data);
         if (returnVal is error) {
-             panic returnVal;
+            error returnValError = returnVal;
```

Comment:
Maybe I am out of context. Is this assignment necessary?

Reply:
Yes, This assignment is necessary

Reply:
You can just cast it.
`panic <error> returnVal`